### ISSUE TITLE
Tcp flow group

### ISSUE THREAD

**Issue by Martin-share** [ViktorAxelsen/TFE-GNN#6]
Hi author, I'm glad you're opening the source code, that's a spirit we should learn from. You mentioned in the md file that only TCP packets are processed, may I ask if you are using splitCap to process the packets to get the following grouping? Thank you for your answer.
![image](https://github.com/ViktorAxelsen/TFE-GNN/assets/93062634/218c239f-0502-4a47-b796-c938a50ed0b2)

**Comment by ViktorAxelsen** [comment]
No, we grouped them ourselves according to the description of the dataset categories on the ISCX dataset website.

**Comment by Martin-share** [comment]
Hi author, nice to hear from you. I am happy that on the iscx-vpn dataset I achieved the following results:. But I still have some problems because the official dataset was modified and the new version of vpn_hangouts_audio2.pcap data downloaded from the official website has the wrong data that can't be cut with splitcap, which may have led to some anomalies in the final result. Can you please provide the data of vpn_hangouts_audio2.pcap that you downloaded at that time conveniently? Thank you. In addition, do you have any suggestion for me about the result shown in the picture? (I'm a novice in machine learning)
![image](https://github.com/ViktorAxelsen/TFE-GNN/assets/93062634/78399517-1aa8-4b78-b235-0f977cf7784a)

**Comment by ViktorAxelsen** [comment]
Great.

You can try to rename "vpn_hangouts_audio2.pcap" as 'vpn_hangouts_audio2.pcapng" and use pcapng2pcap.py to convert this file and try the SplitCap again (In my impression, it seems to work like this. You can try it first).

Aside from fluctuations within a reasonable range, your results look OK. By the way, when you reproduce other baseline models, make sure that the preprocessing method of the dataset is consistent (e.g., dataset partitioning) so that you can get reliable comparison results (preprocessing of network traffic data is relatively complicated, so be careful). From this perspective, just make sure that all your experiments are run on the same dataset (i.e., you can even choose not to use the files in question). 

Hope to see your work in the future!
